### PR TITLE
Add/actions plus guide

### PR DESCRIPTION
**Pull request by joshLong145**
Adds an example for encrypting, then decrypting an `api key` from within an lit action. The flow for this example is:
1. User session is generate from an eth wallet
2. User encrypts encryption key locally, with some `Access Control Conditions`
2. Session, ciphertext, and hash are sent to the lit action for decryption.
5. `api key` is decrypted and used with a provided url for accessing the api the key was intended for

### CODE REVIEW

File: guides/decrypt-api-key-in-action/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    client
+);
+
+console.log("cipher text:", ciphertext, "hash:", dataToEncryptHash);
+
+const res = await client.executeJs({
+    sessionSigs: session,
+    code: genActionSource(url),
+    jsParams: {
+        ciphertext,
```
Comment (DashKash54) [inline guides/decrypt-api-key-in-action/src/index.ts:51]: You're not passing the Access control condition to the Lit Action

File: guides/decrypt-api-key-in-action/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const key = '<your api key>';
+
+let client = new LitNodeClient({
+    litNetwork: 'cayenne'
+});
+await client.connect();
+
+const wallet = genWallet();
+
+const session = await genSession(wallet, client, []);
```
Comment (DashKash54) [inline guides/decrypt-api-key-in-action/src/index.ts:16]: Resources don't include the ability to execute a Lit Action. Does this get added by default?

**Review by DashKash54 — APPROVED** [review]
Left a couple of comments. Looks good otherwise

File: guides/decrypt-api-key-in-action/src/index.ts (hunk 1 above)
Comment (joshLong145) [reply guides/decrypt-api-key-in-action/src/index.ts:64]: good catch, just added them

File: guides/decrypt-api-key-in-action/src/index.ts (hunk 2 above)
Comment (joshLong145) [reply guides/decrypt-api-key-in-action/src/index.ts:17]: no but we do not need any when encrypting, I just updated to include them when we are calling the lit action

File: guides/decrypt-api-key-in-action/src/wallet.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    return new ethers.providers.JsonRpcProvider('https://chain-rpc.litprotocol.com/http');
+}
+
+export const genWallet = () => {
+    // known private key for testing
+    // replace with your own key
+    return new ethers.Wallet('0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80', genProvider());
+}
+
+export const genAuthSig = async (
```
Comment (joshLong145) [inline guides/decrypt-api-key-in-action/src/wallet.ts:16]: @joshLong145 
use this over implementation in pr #8

**Review by spacesailor24 — APPROVED** [review]
🚀 This is awesome, thank you Josh

**Pull request merged**